omnibus/files/private-chef-cookbooks/private-chef/recipes/redis_lb.rb

```diff
@@ -80,9 +82,26 @@
 component_runit_service 'redis_lb'
 
 # Restart the redis_lb runit service.
-runit_service 'redis_lb' do
-  action :restart
-  only_if { is_data_master? }
+ruby_block 'restarting redis_lb' do
+  block do
+    started = false
+    10.times do
```

Comment:
Looks like there is an option to use retries and retry_delay instead of x.times and sleep: https://docs.chef.io/resource_common/#properties